Composer/plugins/azurePublish/src/index.ts

```diff
@@ -195,11 +195,16 @@ class AzurePublisher {
       }
     } catch (error) {
       console.log(error);
+      if (typeof error === 'object' && !(error instanceof Error)) {
```

Comment:
I think these `if` statements should have 3 branches:
- `if (error instanceof Error)` (where we log `error.message`),
- `else if (typeof error === 'object')` where we stringify it, and
- `else`, where we just push it.

The last two are the same as you have them here.

Reply:
To add to this, this logic should be extracted into a single method.

Reply:
done